jobs/deletejob.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * ownCloud - search_lucene
+ *
+ * This file is licensed under the Affero General Public License version 3 or
+ * later. See the COPYING file.
+ *
+ * @author ThomasMüller <deepdiver@owncloud.com>
+ * @copyright Thomas Müller 2015
+ */
+
+namespace OCA\Search_Lucene\Jobs;
+
+use OCA\Search_Lucene\AppInfo\Application;
+use OCA\Search_Lucene\Core\Logger;
+use OC\BackgroundJob\TimedJob;
```

Comment:
we have currently no public base class - not nice but well ... I suspect we need to change these mechanisms anyhow with respect to async processing, queues and so on